src/app/ng-material/ng-material.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {MatIconModule} from '@angular/material/icon';
+import { MatCardModule } from '@angular/material/card';
+import { MatButtonModule } from '@angular/material/button';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    CommonModule
+  ],
```

Comment:
You don't need to import common module here and the declarations section could be deleted.
```suggestion

```